flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcExecutionOptions.java

```diff
@@ -66,12 +76,13 @@ public boolean equals(Object o) {
 		JdbcExecutionOptions that = (JdbcExecutionOptions) o;
 		return batchIntervalMs == that.batchIntervalMs &&
 			batchSize == that.batchSize &&
-			maxRetries == that.maxRetries;
+			maxRetries == that.maxRetries &&
+			parallelism == that.parallelism;
```

Comment:
Will return false when parallelism > 128?

Reply:
Maybe the framework will check whether the parallelism > max parallelism limit.
Moreover where do you get the 128?IIRC,128 is the limit of max keyGroup.

Reply:
My suggestion is that use `equals()` instead of `==` beacause `==` will return false when parallelism's vlue more than 128.

Reply:
ok,I have changed it.